### PR TITLE
Add BigInt to ObjectValue

### PR DESCRIPTION
**Pull request by goodmind**

### CODE REVIEW

**Comment by xyc** [comment]
Nice addition! But CI doesn't seem to pass, and deploy preview doesn't seem to reflect this change either

**Comment by goodmind** [comment]
@xyc no idea why preview doesn't work

**Comment by xyc** [comment]
@goodmind might be build tooling, will take a look this weekend

**Review by xyc — APPROVED** [review]

**Comment by xyc** [comment]
when I run `yarn storybook` with this patch locally, it fails with

```
ERROR in ./stories/object-inspector.js 49:12
Module parse failed: Identifier directly after number (49:12)
You may need an appropriate loader to handle this file type.
| storiesOf('BigInts', module).add('positive', function () {
|   return React.createElement(Inspector, {
>     data: 42n
|   });
| }).add('zero', function () {
 @ ./.storybook/config.js 4:2-40
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js (webpack)-hot-middleware/client.js?reload=true
```

**Comment by goodmind** [comment]
@xyz not sure why, I added babel plugin

**Review by ndelangen — APPROVED** [review]

File: src/object/ObjectValue.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ const ObjectValue = ({ object, styles }) => {
   const mkStyle = key => ({ ...themeStyles[key], ...styles });
 
   switch (typeof object) {
+    case 'bigint':
+      return <span style={mkStyle('objectValueNumber')}>{String(object)}n</span>;
```
Comment (wuweiweiwu) [inline src/object/ObjectValue.js]:
```suggestion
      return <span style={mkStyle('objectValueNumber')}>{String(object)}</span>;
```

**Comment by ndelangen** [comment]
This seems to work?
https://deploy-preview-75--react-inspector.netlify.com/?selectedKind=BigInts&selectedStory=positive&full=0&addons=1&stories=1&panelRight=0

**Comment by goodmind** [comment]
It should have `n` at the end

**Pull request merged**